Library/Homebrew/formula_installer.rb

```diff
@@ -369,8 +369,8 @@ def check_conflicts
 
       $stderr.puts "Please report this to the #{formula.tap} tap!"
       false
-                else
-                  f.linked_keg.exist? && f.opt_prefix.exist?
+    else # rubocop:disable Lint/ElseAlignment
```

Comment:
This should probably be reported as a bug.

Reply:
https://github.com/rubocop-hq/rubocop/issues/6557

Reply:
Thanks @reitermarkus 🙇